packages/documentation/docs/controls/_content-header_styleguide.md

```diff
@@ -0,0 +1,46 @@
+The content header provides a brief overview of the content on a page. It helps the user to understand what the page is about.
+We typically use it at the very top of the page.
+
+![Content header overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2250-4784&mode=design&t=XmCepM9jPR9PImPw-4)
+ 
+1. Back button 
+2. Content headline
+3. Content subtitle
+4. Action buttons
+
+## Variants
+With our content header variants, you can make it easer for the user to follow the hierarchy of the page.
+
+Content header variants:
+- 1: primary - typically used for main pages
+- 2: secondary - typically used for subsections within a page
+
+![Content header variants](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2250-9102&mode=design&t=XmCepM9jPR9PImPw-4)
+
+## Options
+- **HasBackButton**: This option can be enabled if you want to provide a way for the user to navigate back.
+- **HeaderTitle**: Here you can set your page title.
+- **HeaderSubtitle**: If you want to provide additional info for your content you can add a descriptive sentence here.
+- **Variant**: Select primary if you have a page title or secondary if you have a subsection within a page.
+- **Action buttons**: These buttons provide quick access to a content of the page. They enhance the user experience by offering convenient shortcuts for actions that the user might need to perform frequently. Examples could be "Add" or "Edit".
+
+## Behavior 
+- **Interaction**: The back button or action buttons can be triggered by pressing anywhere within the button container.
+- **Alignment**: The content header should always be at the very top left corner at related to the content position. Back button, title and subtitle are automatically aligned within a group at the left side while the action buttons are aligned to a group at the right side.
+- **Cluster action buttons**: The action buttons are automatically aligned to the right. As an example for the primary content header you would have the back button, title and subtitle at the left top corner of the whole page and the action buttons at the right top corner of the page.
+
+## Dos and Don'ts
+- Do use to set your page or subsection header
+- Do use to provide navigation options
+- Do use to provide quick access to common tasks
+- Do use to provide additional context for your content
+- Do use to visualize a clear hierarchy of your page
+- Do use primary content header as page title
+- Do not use a back button when navigating back is not required
+- Do not use action buttons when there are no key actions
```

Comment:
Please avoid double negation - maybe something like "Do not use action buttons for actions that are individual for an object" (bad sentence, but for sure there are use cases where it's not good :P)